src/plots/smith/layout_attributes.js

```diff
@@ -0,0 +1,235 @@
+'use strict';
+
+var colorAttrs = require('../../components/color/attributes');
+var axesAttrs = require('../cartesian/layout_attributes');
+var domainAttrs = require('../domain').attributes;
+var extendFlat = require('../../lib').extendFlat;
+var overrideAll = require('../../plot_api/edit_types').overrideAll;
+
+var axisLineGridAttr = overrideAll({
+    color: axesAttrs.color,
+    showline: extendFlat({}, axesAttrs.showline, {dflt: true}),
+    linecolor: axesAttrs.linecolor,
+    linewidth: axesAttrs.linewidth,
+    showgrid: extendFlat({}, axesAttrs.showgrid, {dflt: true}),
+    gridcolor: axesAttrs.gridcolor,
+    gridwidth: axesAttrs.gridwidth
+}, 'plot', 'from-root');
+
+var axisTickAttrs = overrideAll({
+    tickmode: axesAttrs.tickmode,
+    tickvals: axesAttrs.tickvals,
+    ticktext: axesAttrs.ticktext,
+    ticks: axesAttrs.ticks,
+    ticklen: axesAttrs.ticklen,
+    tickwidth: axesAttrs.tickwidth,
+    tickcolor: axesAttrs.tickcolor,
+    showticklabels: axesAttrs.showticklabels,
+    showtickprefix: axesAttrs.showtickprefix,
+    tickprefix: axesAttrs.tickprefix,
+    showticksuffix: axesAttrs.showticksuffix,
+    ticksuffix: axesAttrs.ticksuffix,
+    showexponent: axesAttrs.showexponent,
+    exponentformat: axesAttrs.exponentformat,
+    minexponent: axesAttrs.minexponent,
+    separatethousands: axesAttrs.separatethousands,
+    tickfont: axesAttrs.tickfont,
+    tickangle: axesAttrs.tickangle,
+    tickformat: axesAttrs.tickformat,
+    tickformatstops: axesAttrs.tickformatstops,
+    layer: axesAttrs.layer
+}, 'plot', 'from-root');
+
+var radialAxisAttrs = {
+    visible: extendFlat({}, axesAttrs.visible, {dflt: true}),
+
+    angle: {
+        valType: 'angle',
+        editType: 'plot',
+        description: [
+            'Sets the angle (in degrees) from which the radial axis is drawn.',
+            'Note that by default, radial axis line on the theta=0 line',
+            'corresponds to a line pointing right (like what mathematicians prefer).',
+            'Defaults to the first `polar.sector` angle.'
```

Comment:
Is the description above correct?

Reply:
See df36ae3a34ca990aace1e236692bfe7a99dc6d76.